src/api/index.ts

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
not urgent, but we should chat about how we want to use index.ts files

Reply:
Very open to adjusting.

In my last project we'd end up with things like `api/api.ts` which was a bit ugly.  I saw this project uses `index.ts` and tried to follow that pattern, but I may have done it wrong or it may not be a precedent we want to continue.

If you let me know specifically what you'd expect this file to be named, I can just change it.

Reply:
let's keep it and make chat in high bandwidth about a plan